Metro-Records/la-metro-councilmatic
# Update search bar hero image

## Overview

This adds the new hero image and deletes the old one.

- Closes #1033

### Demo

#### Before
![image](https://github.com/Metro-Records/la-metro-councilmatic/assets/114717958/7a8d5ade-02e5-4dd2-819b-c47696a969e3)

#### After
![image](https://github.com/Metro-Records/la-metro-councilmatic/assets/114717958/fa1f5187-d387-4873-862c-50795d5dac5f)


## Testing Instructions

 * Navigate to the homepage
 * Confirm that the image displays correctly
